Slim/Player/Protocols/HTTP.pm

```diff
@@ -368,6 +447,133 @@ sub canDirectStreamSong {
 	return $direct;
 }
 
+sub readChunk {
+	my $self  = $_[0];
+	my $v = ${*$self}{'_enhanced'} || return $self->_sysread($_[1], $_[2], $_[3]);
```

Comment:
I don't like `$v` - except when I myself use them and their meaning is totally clear to _me_. 😀

Reply:
see above :smile:

Reply:
Done